src/main/java/uk/gov/hmcts/reform/jobscheduler/services/auth/AuthService.java

```diff
@@ -1,28 +1,22 @@
 package uk.gov.hmcts.reform.jobscheduler.services.auth;
 
-import feign.FeignException;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.validators.ServiceAuthTokenValidator;
 
 @Service
 public class AuthService {
```

Comment:
It looks like this class duplicates `ServiceAuthTokenValidator`. Maybe should be removed?

Reply:
true. did not see prior removing all this. will get on this next thing